src/condor_userlog/condor_check_userlogs.cpp

```diff
@@ -38,18 +38,16 @@ int main(int argc, char **argv)
 	dprintf_set_tool_debug("condor_check_userlogs", 0);
 	set_debug_flags(NULL, D_ALWAYS);
 
-	StringList	logFiles;
+	std::vector<std::string> logFiles;
 	for ( int argnum = 1; argnum < argc; ++argnum ) {
-		logFiles.append(argv[argnum]);
+		logFiles.emplace_back(argv[argnum]);
```

Comment:
Note that

1. std::vector can be initialized with a pair of iterators, the so-called "range constructor"
2. a pointer is-a iterator
3. I think that if the range ctor is called with random-access iterators, there is an implicit "reserve", so there's always one allocation.

So, we can re-write this loop as

```
std::vector<std::string> logFiles {&argv[1], &argv[argc};
```

Reply:
I like it. Consider it done.